notebooks/community/vizier/conversions_vertex_vizier_and_open_source_vizier.ipynb

```diff
@@ -0,0 +1,624 @@
+        "\n",
+        "\n",
+        "def CreateMetrics(r, theta):\n",
+        "    # Evaluate both objective metrics for this trial\n",
+        "    y1 = Metric1Evaluation(r, theta)\n",
+        "    y2 = Metric2Evaluation(r, theta)\n",
+        "    print(\n",
+        "        \"[r = {}, theta = {}] => y1 = r*sin(theta) = {}, y2 = r*cos(theta) = {}\".format(\n",
+        "            r, theta, y1, y2\n",
+        "        )\n",
+        "    )\n",
+        "    measurement = pyvizier.Measurement()\n",
+        "    measurement.metrics[\"y1\"] = y1\n",
+        "    measurement.metrics[\"y2\"] = y2\n",
+        "\n",
+        "    # Return the results for this trial\n",
+        "    return measurement"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "2DgUIEpZ-_fJ"
+      },
+      "source": [
+        "### Optimization\n",
+        "\n",
+        "The following code defines a study with parameters and metrics. Then evaluates the metric information based on the suggestions from Vizier and reports the metrics value back. After a few round of iteration, you can get optimal trials by calling optimal_trials(). The code is adapt on both Vertex Vizier and OSS Vizier."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "s-AHfPOASXXW"
+      },
+      "outputs": [],
+      "source": [
+        "problem = pyvizier.StudyConfig()\n",
+        "problem.algorithm = pyvizier.Algorithm.RANDOM_SEARCH\n",
+        "\n",
+        "# Objective Metrics\n",
+        "problem.metric_information.append(\n",
+        "    pyvizier.MetricInformation(name=\"y1\", goal=pyvizier.ObjectiveMetricGoal.MINIMIZE)\n",
+        ")\n",
+        "problem.metric_information.append(\n",
+        "    pyvizier.MetricInformation(name=\"y2\", goal=pyvizier.ObjectiveMetricGoal.MAXIMIZE)\n",
+        ")\n",
+        "\n",
+        "# Defines the parameters configuration.\n",
+        "root = problem.search_space.select_root()\n",
+        "root.add_float_param(\"r\", 0, 1.0, scale_type=pyvizier.ScaleType.LINEAR)\n",
+        "root.add_float_param(\"theta\", 0, 1.57, scale_type=pyvizier.ScaleType.LINEAR)\n",
+        "\n",
+        "STUDY_DISPLAY_NAME = \"{}_study_{}\".format(\n",
+        "    PROJECT_ID.replace(\"-\", \"\"), datetime.datetime.now().strftime(\"%Y%m%d_%H%M%S\")\n",
+        ")\n",
+        "study = CreateStudy(\n",
+        "    project=PROJECT_ID,\n",
+        "    location=REGION,\n",
+        "    display_name=STUDY_DISPLAY_NAME,\n",
+        "    problem=problem,\n",
+        ")\n",
+        "\n",
+        "for _ in range(MAX_NUM_ITERATIONS):\n",
+        "    trials = study.suggest(count=SUGGESTION_COUNT)\n",
+        "    for trial in trials:\n",
+        "        materialize_trial = trial.materialize()\n",
+        "        measurement = CreateMetrics(\n",
+        "            materialize_trial.parameters.get_value(\"r\"),\n",
```

Comment:
You can get the trial parameters as a dict using StudyConfig.trial_parameters(trial).

So this can become:
params = problem.trial_parameters(trial)
measurement = CreateMetrics(params["r"], params["theta"])

Reply:
When I tried the OSS Vizier, I got the issue. So I use the get_value("r")  as temp solution. 

study TrialIterable(_iterable_factory=<function Study.trials.<locals>.iterable_factory at 0x7f0750140e50>, _client=VizierClient(_study_resource_name='owners/halio/studies/cloudmle2etesting_study_20221024_123140', _client_id='Unused client id.'))
Traceback (most recent call last):
  File "/home/halio/Program/sample/test.py", line 80, in <module>
    measurement = CreateMetrics(
  File "/home/halio/Program/sample/test.py", line 36, in CreateMetrics
    y1 = Metric1Evaluation(r, theta)
  File "/home/halio/Program/sample/test.py", line 25, in Metric1Evaluation
    return r * math.sin(theta)
TypeError: must be real number, not ParameterValue